src/main/java/emissary/core/IBaseDataObjectHelper.java

```diff
@@ -37,15 +37,27 @@ private InternalIdBaseDataObject(final UUID internalId) {
 
     private IBaseDataObjectHelper() {}
 
+    /**
+     * Clones an IBaseDataObject.
+     * 
+     * @param iBaseDataObject the IBaseDataObject to be cloned.
+     * @return the clone of the IBaseDataObject passed in.
+     */
+    public static IBaseDataObject clone(final IBaseDataObject iBaseDataObject) {
+        return clone(iBaseDataObject, true);
+    }
+
     /**
      * Clones an IBaseDataObject equivalently to emissary.core.BaseDataObject.clone(), which duplicates some attributes.
      * 
      * A "fullClone" duplicates all attributes.
      * 
+     * @deprecated prefer {@link #clone(IBaseDataObject)}
      * @param iBaseDataObject the IBaseDataObject to be cloned.
      * @param fullClone specifies if all fields should be cloned.
      * @return the clone of the IBaseDataObject passed in.
      */
+    @Deprecated(forRemoval = true)
```

Comment:
```suggestion
    @Deprecated
```
This makes it so the build doesn't warn on itself for this change.